docs/python/reference/configuration.md

```diff
@@ -24,29 +24,34 @@ class DBOSConfig(TypedDict):
     name: str
     database_url: Optional[str]
     sys_db_name: Optional[str]
-    app_db_pool_size: Optional[int]
     sys_db_pool_size: Optional[int]
     db_engine_kwargs: Optional[Dict[str, Any]]
     log_level: Optional[str]
     otlp_traces_endpoints: Optional[List[str]]
-    run_admin_server: Optional[bool]
+    otlp_logs_endpoints: Optional[List[str]]
     admin_port: Optional[int]
+    run_admin_server: Optional[bool]
 ```
 
 - **name**: Your application's name.
-- **database_url**: A connection string to a Postgres database. The supported format is
-```
-postgresql://[username]:[password]@[hostname]:[port]/[database name]
-```
-The `sslmode=require`, `sslrootcert` and `connect_timeout` parameter keywords are also supported.
-Defaults to:
+- **database_url**: A connection string to a Postgres database. DBOS will use the connection string, unmodified, to create a [SQLAlchemy engine](https://docs.sqlalchemy.org/en/20/core/engines.html).
+
+:::info
+Passwords in connection strings should be escaped (e.g., with [urllib](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote)).
+:::
+
+:::info
+DBOS Transact requires the [psycopg 3](https://www.psycopg.org/psycopg3/docs/) driver and will accordingly set the driver to `postgresql+psycopg` before creating an engine. If you use [alembic](https://alembic.sqlalchemy.org/en/latest/) to manage database migrations, you'll need to set the driver in your connection string.
+:::
+
+I none is provided, DBOS will use this default connection string:
 ```
-postgresql://postgres:dbos@localhost:5432/[application name]
+postgresql://postgres:${PGPASSWORD}@localhost:5432/application_name?connect_timeout=10&sslmode=prefer
 ```
-- **sys_db_name**: Name for the [system database](../../explanations/system-tables) in which DBOS stores internal state. Defaults to `{database name}_dbos_sys`.
-- **app_db_pool_size**: The size of the connection pool used by [transactions](../tutorials/transaction-tutorial.md) to connect to your application database. Defaults to 20.
-- **sys_db_pool_size**: The size of the connection pool used for the [DBOS system database](../../explanations/system-tables). Defaults to 20.
-- **db_engine_kwargs**: Additional keyword arguments passed to SQLAlchemy’s [`create_engine()`](https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine), applied to both the application and system database engines. Defaults to:
+
+If `PGPASSWORD` is not exported, the default password will be `dbos`. The database name will be derived from your application's name.
```

Comment:
Can you change this back to what it was before?  Much clearer I think. Also you don't need to specify sslmode=prefer, that's the default.

Reply:
Before we were supporting an incomplete format (w/o query parameters), that's why we needed to break down the format we support. Now we support arbitrary connection strings.